jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenterTest.java

```diff
@@ -259,4 +277,71 @@ public void testActiveSearchFiltersProcessInstanceId(){
 
     }
 
+    @Test
+    public void testGoToJobErrorSummary() {
```

Comment:
I find the test name confusing. 
The current name implies testing navigation to the error (in Execution errors perspective) rather than from the error to the erroneous job (in Jobs) perspective.

Could we rename this test and the next one which tests navigation to Process Instances to something along the lines of _testGoToJobPerspective_ or _testGoToJobs_?